.gitmodules

```diff
@@ -32,7 +32,7 @@
 	ignore = dirty
 [submodule "3rdparty/hidapi"]
 	path = 3rdparty/hidapi
-	url = https://github.com/RPCS3/hidapi
+	url = https://github.com/libusb/hidapi.git
```

Comment:
Needs a good justification because it has custom commits from RPCS3 devs...

Reply:
This discussion has already been had on discord: https://discordapp.com/channels/272035812277878785/272083901277667342/739462823570309132

Reply:
Thanks @JohnHolmesII 
Summary is:
RPCS3 specific customisations related to CMake, and Windows support for Rumble/Lights.
Rumble/Lights were tested by me as working fine under master libusb/hidapi now.
CMake I had wrongly assumed was 'ok', because it built fine for me in Windows MSVC, but it seems that isn't using CMake.
I'm pushing with libusb/hidapi to get their CMake support into master, at which time I'll give this another shot.